### DISCUSSION TITLE
Sleeping and turning off display settings

### DISCUSSION THREAD

**Discussion by FrancescoRagusa03** [seerge/g-helper#2747] [Ideas]
Hi I own a asus tuf a15, which has some problems when it comes to the  battery management: the settings for turning off and suspend the pc in the control panel don't work.
I use g helper from a bit and I find really helpful the option to hybernate the pc after tot minutes. So I was wondering if you could insert in the program more settings like that. thanks

**Comment by seerge** [comment]
@MacCutcheon hello, Sleep and Power off is handled by Windows not by apps :) If such basic functionality is broken in your system I would try to troubleshoot that first
